### ISSUE TITLE
Previewing and then canceling does not produce a warning that content will be lost

### ISSUE THREAD

**Issue by erik-alm** [splitbrain/dokuwiki#3711]
I don't know if this is a conscious design choice or a bug.

In Hogfather, when I edit a page, then preview it, I am able to press cancel and the wiki will discard my edit without showing the verification dialog you usually get when canceling an edited page.

I'd like to always get that confirmation dialog since in the majority of cases I like to save or keep editing after a preview and would not want to accidentally press the cancel button and lose all my work.

**Comment by Michaelsy** [comment]
Sorry, I have checked again and have also been able to reproduce the bug.

Works in Greebo
Does not work in Hogfather and Igor (RC)

\- Michael Sy.

**Comment by erik-alm** [comment]
I've verified it works like this on macOS in Firefox, Chrome, and Safari.

**Comment by splitbrain** [comment]
Yeah, this seems to be broken since a while. I fixed it in a branch but this won't go into Igor anymore.
